persistence/binary/src/main/java/one/microstream/persistence/binary/util/PersistenceExceptionTypeImportTypeNotFound.java

```diff
@@ -0,0 +1,32 @@
+package one.microstream.persistence.binary.util;
+
+/*-
+ * #%L
+ * MicroStream Persistence Binary
+ * %%
+ * Copyright (C) 2019 - 2022 MicroStream Software
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import one.microstream.persistence.exceptions.PersistenceException;
+
+@SuppressWarnings("serial")
+public class PersistenceExceptionTypeImportTypeNotFound extends PersistenceException
```

Comment:
Should be called `BineraPersistenceExceptionTypeImportTypeNotFound`, extends from `BinaryPersistenceException` and be in package `one.microstream.persistence.binary.exceptions`.

Reply:
ok, changed